app/assets/stylesheets/catarse_bootstrap/_extra.sass

```diff
@@ -13,6 +13,8 @@ img, iframe
 .list-answer
   display: none
 
+.project-search-image
+  max-width: 50px !important
```

Comment:
the !important in the img was added by @vicnicius here: 265eba603d89b6ccae6fa1d6c5384a711466e692

Maybe this rule could be more specific. I believe this has to do with redactor and the use of imager embeds.